### DISCUSSION TITLE
Help plz

### DISCUSSION THREAD

**Discussion by EmpoweredPhoenix** [3kh0/ext-remover#435] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
can you make a new point blank method for lightspeed filter agent.

**Comment by obkduishinijf** [comment]
point blank works on all extensions, because it constantly reloads them.

**Reply by Aka-but-nice** [reply]
He's not talking about the old point blank, he means the new one, which in my opinion is better, and I'm sure anyone can say the same. Since there is no xss found yet, in lightspeed, you'll have to deal with old point blank, you could also try killcurly

**Comment by piplup7575** [comment, marked as answer]
@Aka-unblock killcurly will not work with disabling lightspeed
![image](https://user-images.githubusercontent.com/88395302/215272973-f49d22eb-011f-48df-b95c-9e3902eb69ca.png)
use inspect ltbeef or point blank for now

**Reply by Aka-but-nice** [reply]
Good to know, I will update the read me, if you can that would be great, I honestly don't have the time.
